geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/RegionManagementDunitTest.java

```diff
@@ -79,6 +79,7 @@ public void createsRegion() throws Exception {
 
   @Test
   public void createsAPartitionedRegionByDefault() throws Exception {
+    IgnoredException.addIgnoredException("cache element orders already exists.");
```

Comment:
We should not be dumping exceptions like this into the log. We should not even need to log anything.

Reply:
agree